packages/esm-patient-forms-app/src/forms/use-launch-forms-workspace.ts

```diff
@@ -0,0 +1,29 @@
+import { useMemo } from 'react';
+import { usePatient } from '@openmrs/esm-framework';
+import {
+  launchPatientWorkspace,
+  launchStartVisitPrompt,
+  useVisitOrOfflineVisit,
+} from '@openmrs/esm-patient-common-lib';
+
+function useLaunchFormsWorkspace() {
+  const { patientUuid } = usePatient();
+  const { currentVisit } = useVisitOrOfflineVisit(patientUuid);
+
+  const results = useMemo(
+    () => ({
+      launchFormsWorkspace: () => {
+        if (currentVisit) {
+          launchPatientWorkspace('clinical-forms-workspace');
```

Comment:
Eventually, we shoudl probably refactor things so you can just call `launchPatientWorkspace(..., { requireVisit: true });` and get this for free.